tool/SignalGenerator/signalgenerator.h

```diff
@@ -39,6 +39,13 @@ namespace daq::streaming_protocol::siggen {
     static const char EXECUTIONTIME[] = "executionTime";
     static const char PROCESSPERIOD[] = "processPeriod";
 
+    static const char RANGE[] = "range";
```

Comment:
Could you please use the existing constant from
"include/streaming_protocol/Defines.h"?

Reply:
Done